as_csv.gemspec

```diff
@@ -14,13 +14,13 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = %w[lib]
 
-  gem.add_runtime_dependency "activemodel", '>= 3.0'
-  gem.add_runtime_dependency "actionpack", '>= 3.0'
+  gem.add_runtime_dependency "activemodel", '>= 7.0'
+  gem.add_runtime_dependency "actionpack", '>= 7.0'
   gem.add_runtime_dependency "responders"
```

Comment:
If we're going to drop the responders feature (which seems reasonable since we don't actually integrate with the feature), should we drop this dependency?